app/Repositories/Behaviors/HandleApiRelations.php

```diff
@@ -69,7 +70,18 @@ public function updateMultiBrowserApiRelated($object, $fields, $relationship, $t
         ])->delete();
 
         $position = 1;
+
         collect($relatedElements)->each(function ($values) use ($relationship, &$position, $object) {
+            // if ($relationship == 'multi_browser' && $values['endpointType'] == 'exhibitions') {
+            //     dd([
+            //         'subject_id' => $object->getKey(),
+            //         'subject_type' => $object->getMorphClass(),
+            //         'related_id' => $values['id'],
+            //         'related_type' => $values['endpointType'],
+            //         'browser_name' => $relationship,
+            //         'position' => $position,
+            //     ]);
+            // }
```

Comment:
looks like some test code got left in here^

Reply:
Ah! Good catch. Fixed! 🔧